src/poetry_plugin_pypi_proxy/plugin.py

```diff
@@ -106,8 +106,11 @@ def activate(self, poetry: Poetry, io: IO) -> None:
         )
 
         # If this is a publish command to Pypi, we'll silenly redirect to the proxy
-        if io.input.arguments["command"] == "publish" and not io.input.option(
-            "repository"
-        ):
+        if io.input.arguments["command"] == "publish" and not io.input.option("repository"):
             io.input.set_option("repository", proxy_id)
             poetry.config._config["repositories"] = {proxy_id: {"url": proxy_url}}
+
+  
+    # Removf suffix from url appropriately so it can be used by the proxy object
```

Comment:
Typo here, can this be changed to a docstring? First line after the def surrounding by triple quotes. Ex:

    def parse_url(url: str) -> str:
        """Remove suffix from url appropriately so it can be used by the proxy object."""
        ...